### PR TITLE
Sign up and VOMS

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Try to get extract common stuff
- Sign up section
- Move VOMS info to HTC getting started

----

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
The Getting Started section of HTC seems redundant with the changes so I completely removed it.
The workload manager docs should include the documentation that is missing from there

**Comment by enolfc** [comment]
I'm missing some stuff, fixing it

**Review by vardizzo-lab — DISMISSED** [review]
Further insights must be entered here to explain the use of VO access, the relationship between a VO and a COU group, the role of Perun between VOMS and Check-in, finally missing the link to proc24 (how to register a VO) in the paragraph of the VO / VOMS.

File: content/en/users/check-in/signup/_index.md
Comment (andrea-manzi) [inline content/en/users/check-in/signup/_index.md:10]:
```suggestion
You need to sign up for an account for accessing the EGI services. As part of
```

File: content/en/users/getting-started/_index.md
Comment (andrea-manzi) [inline content/en/users/getting-started/_index.md:12]:
```suggestion
You need to sign up for an account for accessing the EGI services. This process
```

File: content/en/users/getting-started/_index.md
Comment (andrea-manzi) [inline content/en/users/getting-started/_index.md:14]:
```suggestion
your existing  credential (for example using an eduGAIN IdP) with EGI.
```

**Comment by andrea-manzi** [comment]
we should use the same way on the documentation to refer to X509 certificates, cause sometimes we use X509 others X.509 or x509.

File: content/en/users/htc/_index.md
Comment (andrea-manzi) [inline content/en/users/htc/_index.md:100]:
```suggestion
* [Data Transfer Service (FTS)](../data-transfer/)
```

File: content/en/users/htc/_index.md
Comment (andrea-manzi) [inline content/en/users/htc/_index.md:108]:
```suggestion
Access to HTC resources is based on [Virtual Organisations](../getting-started/#virtual-organisations-vos),
```

File: content/en/users/getting-started/_index.md
Comment (gwarf) [reply content/en/users/getting-started/_index.md:14]: You left one space :)

Comment (gwarf) [reply content/en/users/getting-started/_index.md:14]: I edited your suggestions 😈

**Comment by enolfc** [comment]
> we should use the same way on the documentation to refer to X509 certificates, cause sometimes we use X509 others X.509 or x509.

I have moved everything to `X.509`

**Comment by enolfc** [comment]
> Further insights must be entered here to explain the use of VO access, the relationship between a VO and a COU group, the role of Perun between VOMS and Check-in, finally missing the link to proc24 (how to register a VO) in the paragraph of the VO / VOMS.

where should this be added?
PROC24 is linked in the VOMS page. 

Relationship between COU group and VO and Perun/VOMS is relevant for Check-in documentation, but probably out of scope for a Getting Started page

**Comment by enolfc** [comment]
I also brought the whole https://wiki.egi.eu/wiki/USG_Getting_Certificate#Requesting_the_Certificate to the VOMS page

**Review by andrea-manzi — DISMISSED** [review]
thanks!

**Comment by enolfc** [comment]
> thanks!

@andrea-manzi I have detected some duplication in your bits... you will need to re-approve soon ;)

File: content/en/users/_index.md
Comment (andrea-manzi) [inline content/en/users/_index.md:17]:
```suggestion
You can ask for more information about the services at: [here](https://www.egi.eu/more-information/)
```

File: content/en/users/check-in/voms/_index.md
Comment (andrea-manzi) [inline content/en/users/check-in/voms/_index.md:77]: why this is pointing to CERN?

Comment (enolfc) [reply content/en/users/check-in/voms/_index.md:77]: I don't know, it was in the docs provided by @YinChenEGI. I took a look and those docs look quite complete. Do you have a better alternative.

Comment (YinChenEGI) [reply content/en/users/check-in/voms/_index.md:77]: I get from the eNMR VO HTC user document:  https://ri-vis.eu/compute/support/egi-platforms/vo-registration-troubleshooting. CERN has quite complete documentation and example for Certificate installation.

Comment (gwarf) [reply content/en/users/check-in/voms/_index.md:77]: I don't know a better source for this information.
We could plan to document it later (like by recording an issue in GitHub) but I feel it may OK for now to keep this link.
@andrea-manzi don't be afraid CERN is a trusted source ;)

Comment (gwarf) [reply content/en/users/check-in/voms/_index.md:77]: We have things like https://wiki.egi.eu/wiki/USG_Getting_Certificate, but the CERN page may be enough for now

Comment (andrea-manzi) [reply content/en/users/check-in/voms/_index.md:77]: Hey, sorry i didn't want to start a long discussion:-) i just was wondering why were pointing to a particular CA instructions, and of course i agree that is quite good documentation:-P

**Review by gwarf — APPROVED** [review]
LGTM, want to see this merged.

**Review by andrea-manzi — APPROVED** [review]
good for me to merge

**Comment by gwarf** [comment]
I'm a bit struggling being sure who GitHub is willing a review from...
@vardizzo-lab it seems you may be the one with the power to validate this, can you have a look?

**Comment by enolfc** [comment]
> I'm a bit struggling being sure who GitHub is willing a review from...
> @vardizzo-lab it seems you may be the one with the power to validate this, can you have a look?

GitHub says: Waiting on code owner review from @vardizzo-lab and/or @CatalinCondurache.

**Comment by gwarf** [comment]
> GitHub says: Waiting on code owner review from @vardizzo-lab and/or @CatalinCondurache.

Where do you see this? from GitHub CLI?

**Comment by gwarf** [comment]
> > GitHub says: Waiting on code owner review from @vardizzo-lab and/or @CatalinCondurache.
> 
> Where do you see this? from GitHub CLI?

So it wasn't my eyes but a GitHub bug, it's clear now :)

**Review by vardizzo-lab — APPROVED** [review]

**Review by vardizzo-lab — APPROVED** [review]
OK.

**Comment by vardizzo-lab** [comment]
Reviewed and approved.

**Comment by gwarf** [comment]
Thanks, but please don't close it so that we can merge it ;)

**Comment by gwarf** [comment]
@CatalinCondurache it seems we also need your approval, probably as it's touching parts of the doc that are under your responsibility, can you please have a look?

**Review by CatalinCondurache — APPROVED** [review]
It looks good.

**Review by andrea-manzi — APPROVED** [review]

**Review by gwarf — APPROVED** [review]
LGTM

**Review by vardizzo-lab — APPROVED** [review]

**Pull request merged**
